src/virtual/models/item.ts

```diff
@@ -13,6 +15,7 @@ export default class Item {
     type: ItemType;
     use?: CallableFunction;
     count?: number;
+    chance?: number;
```

Comment:
What is an item chance?

Reply:
@OlaOpdahl How is this resolved?

Reply:
Item chance from dropTableItem, removed it from Item.